redash/handlers/__init__.py

```diff
@@ -32,6 +32,14 @@ def status_api():
     return jsonify(status)
 
 
+@routes.app_context_processor
+def inject_variables():
+    return dict(
+        name=settings.NAME,
+        logoUrl=settings.LOGO_URL,
```

Comment:
`logoUrl` -> `logo_url`
(we're using `camelCase` only for Javascript and `snake_case` for Python)